packages/oui-navbar/src/navbar.controller.js

```diff
@@ -1,52 +1,93 @@
 import { addBooleanParameter } from "@oui-angular/common/component-utils";
+import get from "lodash/get";
 
 export default class {
-    constructor ($attrs, $document, $element, $timeout, ouiNavbarConfiguration, NavbarService, KEYBOARD_KEYS) {
+    constructor ($attrs, $document, $element, $timeout, ouiNavbarConfiguration, KEYBOARD_KEYS) {
         "ngInject";
 
         this.$attrs = $attrs;
         this.$document = $document;
         this.$element = $element;
         this.$timeout = $timeout;
         this.config = ouiNavbarConfiguration;
-        this.navbarService = NavbarService;
+
+        this.keyboardNav = {};
         this.KEYBOARD_KEYS = KEYBOARD_KEYS;
     }
 
+    addItemToGroup (item, groupName) {
+        // Create the group if it doesn't exist
+        if (angular.isUndefined(this.keyboardNav[groupName])) {
```

Comment:
`this.keyboardNav[groupName] === undefined` ?

Reply:
Yes, it's the same :)